Ports/gpgme/package.sh

```diff
@@ -18,3 +18,7 @@ configopts=(
     "--with-libassuan-prefix=${SERENITY_INSTALL_ROOT}/usr/local"
     "--with-sysroot=${SERENITY_INSTALL_ROOT}"
 )
+
+post_install() {
+    run sed -i 's#/usr/local#${CMAKE_SYSROOT}/usr/local#g' ${SERENITY_INSTALL_ROOT}/usr/local/lib/cmake/Gpgmepp/GpgmeppConfig.cmake
+}
```

Comment:
I hate to be a pedant, but surely this can be a proper patch against `lang/cpp/src/GpgmeppConfig.cmake.in.in` in the source tree, right? Then we can also look into upstreaming the patch some time in the future.